frontend/components/main.jsx

```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 
-const MiddleLink = styled(Link)`
-  position: absolute;
-  top: ${(tableSize / 2) - (middleLockSize / 2) - (xsSpacing / 2)}px;
-  left: ${(tableSize / 2) - (middleLockSize / 2) - (xsSpacing / 2)}px;
+const Person = styled.img.attrs(props => ({
+  src: props.src,
+}))`
+  width: ${pictureSize}px;
+  height: ${pictureSize}px;
+  border-radius: 50%;
 `;
+// border: 2px solid black;
 
 const Container = styled.div`
-  position: absolute;
-  width: 50%;
-  top: ${(tableSize / 2) - (lockSize / 2)}px;
-  right: 0;
-  text-align: right;
-  transform: rotate(${props => props.rotation}deg);
-  transform-origin: left;
-`;
-
-const Content = styled.div`
   display: flex;
-  flex-direction: row;
-  justify-content: flex-end;
+  flex-direction: column;
+  justify-content: space-evenly;
   align-items: center;
+  height: 130px;
+  width: 100px;
+  background-color: ${whiteLilac}
+  border-radius: 5px;
+  margin: 5px;
 `;
-
-const Line = styled.div`
-  border: 1px solid black;
-  width: ${(tableSize / 2) - lockSize - (xsSpacing * 2) - (middleLockSize / 2) - (xsSpacing * 2)}px;
-  pointer-events: none;
-`;
+// border: 2px solid ${tamarind};
 
 export default class Main extends React.Component {
   render() {
-    const items = [
-      { to: PEOPLE['tim'].path, key: 1, src: PEOPLE['tim'].src },
-      { to: PEOPLE['jay'].path, key: 2, src: PEOPLE['jay'].src },
-      { to: PEOPLE['ryan'].path, key: 3, src: PEOPLE['ryan'].src },
-      { to: PEOPLE['kristi'].path, key: 4, src: PEOPLE['kristi'].src },
-      { to: PEOPLE['erica'].path, key: 5, src: PEOPLE['erica'].src },
-      { to: PEOPLE['maryann'].path, key: 6, src: PEOPLE['maryann'].src },
-      { to: PEOPLE['helena'].path, key: 7, src: PEOPLE['helena'].src },
+    const firstRow = [
+      { name: PEOPLE['tim'].displayName, to: PEOPLE['tim'].path, key: 1, src: PEOPLE['tim'].src },
+      { name: PEOPLE['jay'].displayName, to: PEOPLE['jay'].path, key: 2, src: PEOPLE['jay'].src },
+      { name: PEOPLE['ryan'].displayName, to: PEOPLE['ryan'].path, key: 3, src: PEOPLE['ryan'].src },
+    ];
+
+    const secondRow = [
+      { name: PEOPLE['matt'].displayName, to: PEOPLE['matt'].path, key: 4, src: PEOPLE['matt'].src },
+      { name: PEOPLE['kristi'].displayName, to: PEOPLE['kristi'].path, key: 5, src: PEOPLE['kristi'].src },
+    ];
+
+    const thirdRow = [
+      { name: PEOPLE['erica'].displayName, to: PEOPLE['erica'].path, key: 6, src: PEOPLE['erica'].src },
+      { name: PEOPLE['maryann'].displayName, to: PEOPLE['maryann'].path, key: 7, src: PEOPLE['maryann'].src },
+      { name: PEOPLE['helena'].displayName, to: PEOPLE['helena'].path, key: 8, src: PEOPLE['helena'].src },
     ];
 
     return (
       <div>
         <WelcomeHeader {...this.props} />
         <Table>
           {
-            items.map((item, index) => {
-              const rotation = index * (360 / items.length);
+            firstRow.map((item) => {
+              return (
+                <StyleLink to={item.to} key={item.key}>
```

Comment:
these all look the same - move to a `renderPerson` method? makes the jsx really simple:
```
<Table>
  {firstRow.map(this.renderPerson)}
</Table>
```

Reply:
I like this idea. I'll implement a renderPerson method